docs/targets/tizenrt/artik05x/IoT.js-API-artik053.md

```diff
@@ -0,0 +1,31 @@
+## Artik053 module
+
+## `Pin`
+To use system IO, such as GPIO, PWM and ADC you must know pin name.
```

Comment:
will it be OK if I change it to:
PWM (ADC, GPIO, etc in the future) ? I think we should have all pins described this way. It is easy and nice for external developers. 0 for PWM and 0 for GPIO is the same number but differend PIN. I think more readable is to have pin.PWM0 and pin.GPIO0